src/Filter/FilterComposite.php

```diff
@@ -25,11 +25,9 @@ final class FilterComposite implements FilterInterface
      */
     public const CONDITION_AND = 2;
 
-    /** @var ArrayObject */
-    protected $andFilter;
+    protected ArrayObject $andFilter;
```

Comment:
Careful with `protected` stuff :|

Reply:
it is final class, and doesn't has parent property usage.

Reply:
Oooh, didn't see - should we change these to `private`, when we encounter them? :thinking:

Reply:
yes, updated to private https://github.com/laminas/laminas-hydrator/pull/91/commits/93f4af2165b1b2788372beedd6bb5f95ed8371ed